docs/docarray-v2.md

```diff
@@ -115,12 +113,27 @@ with Deployment(uses=MyExec) as dep:
 ```
 
 
+## Compatible Features
+
+Jina is working to offer full compatibility of the Jina framework with the power of the new DocArray version.
+
+At this moment, these are the features supported if the APIs described in the previous sections are used.
+
+- All the features offered by {ref}`Deployment <deployment>` where a single Executor is served and {ref}`scaled <scale-out>`. This support includes the usage of both
+HTTP and gRPC protocols and the usage of both of them at the same type.
+
+- When combining multiple Deployments in a pipeline using a {ref}`Flow <flow-cookbook>`, multiple limitations exist at this moment:
+
+    - Only gRPC protocol is supported.
+    - Only linear Flows are supported, no topologies using bifurcations can be used at the moment.
+    - There must exist compatibility between the output schema of an Executor and the input schema of the next one.
```

Comment:
```suggestion

````{admonition} Note
:class: note

With DocArray 0.30 support, Jina introduced the concept of input/output schema at the Executor level. In order to chain multiple Executor into a Flow you always need to make sure that the output schema of an Executor is the same as the Input of the Executor that follows him in the Flow
````